client/scss/components/_button.scss

```
@@ -151,6 +151,62 @@
 
     &.button-secondary {
       border-color: theme('colors.black-20');
+      color: $color-button;
+      background-color: transparent;
+      border: 1px solid $color-button;
+
+      .icon-wrapper {
+        border-right: 1px solid $color-button;
+        background-color: transparent;
+      }
+
+      &:hover {
+        background-color: theme('colors.secondary.50');
+        color: $color-button;
+      }
+
+      &:disabled,
+      &[disabled] {
+        background-color: transparent;
+        border-color: $color-grey-3;
+        color: $color-grey-3;
+        cursor: default;
+        pointer-events: none;
+
+        .icon-wrapper {
+          border-right: 1px solid $color-grey-3;
+          background-color: transparent;
+        }
+      }
+
+      &.button-small {
+        border: none;
+        padding: 0 30px 30px;
+
+        &:hover {
+          background-color: transparent;
+          color: $color-button;
+        }
+
+        .icon-wrapper {
+          border: 2px solid $color-button;
+          border-radius: 50%;
+          border-color: $color-button;
+          background-color: transparent;
+          height: fit-content;
+          width: fit-content;
+
+          &:hover {
+            background-color: $color-button;
+            color: $color-white;
+          }
+        }
```

Several things I'm going to roll-up into one suggestion here, because this whole block needs some restructuring:

- We don't need to define the hover styles for the main part of the button if they are not changing from the base button.
- The icon needs to change when hovering on the text, as well, not just hovering the icon itself.
- It looks to me in the original design like the border width should be 1px.
- If you're defining the border color in the `border` shorthand property, you don't need another `border-color` property.
- I think it's preferable to have all the hover styles together.
- I didn't know about the `fit-content` values! That's very cool!

Here's how I would handle all of the above:

```suggestion
        .icon-wrapper {
          border: 1px solid $color-button;
          border-radius: 50%;
          height: fit-content;
          width: fit-content;
        }

        &:hover {
          background-color: transparent;

          .icon-wrapper {
            background-color: $color-button;
            color: $color-white;
          }
        }
```